octoprint_mrbeam/templates/quick_text.jinja2

```diff
@@ -5,11 +5,12 @@
     </div>
     <div class="modal-body">
         <form role="form" class="form-search" data-bind="submit: function(){$('#quick_text_dialog').modal('hide');}" _lpchecked="1">
-            <input type="text" id="quick_text_dialog_text_input" class="input-block quicktext-input" data-bind="textInput: currentQuickText" placeholder="QuickText..." autocomplete="off">
+            <input type="text" id="quick_text_dialog_text_input" class="input-block quicktext-input" data-bind="textInput: currentQuickText" placeholder="{{ _('QuickText...') }}" autocomplete="off">
                 <div class="row-fluid padding-top">
-                    <div class="btn btn-small span3 quicktext-font-button" data-bind="click: currentQuickTextFontPrev"><i class="icon-chevron-left" title="Previous Font"></i> Aa </div>
+                    {# Translators: Aa - the first alphabet letter capitalized and not #}
+                    <div class="btn btn-small span3 quicktext-font-button" data-bind="click: currentQuickTextFontPrev"><i class="icon-chevron-left" title="{{ _('Previous Font') }}"></i> {{ _('Aa') }} </div>
                     <div class="span6 quicktext-fontname" id="quick_text_dialog_font_name"></div>
-                    <div class="btn btn-small span3 quicktext-font-button" data-bind="click: currentQuickTextFontNext"> Aa <i class="icon-chevron-right" title="Next Font"></i></div>
+                    <div class="btn btn-small span3 quicktext-font-button" data-bind="click: currentQuickTextFontNext"> {{ _('Aa') }} <i class="icon-chevron-right" title="{{ _('Next Font') }}"></i></div>
```

Comment:
Not sure if the Aa needs to be translated. At most for a language with different characters.

This is how this string shows (on the buttons):
<img width="574" alt="screenshot 2019-03-07 19 08 53" src="https://user-images.githubusercontent.com/4582388/53978837-7895b180-410c-11e9-9eb4-198eaedbdddd.png">

Reply:
this is ok